Bug: Map symbol missing in Plugin Class

Caught the following error while `flutter build apk` .
```
Initializing gradle...                                              1.4s
Resolving dependencies...                                          20.3s
/home/harsh/.pub-cache/hosted/pub.dartlang.org/flutter_freshchat-1.1.2/android/src/main/java/com/freshchat/flutter_freshchat/FlutterFreshchatPlugin.java:83: error: cannot find symbol
            final Map<String, String> customProperties = call.argument("custom_property_list");
                  ^                                                     
  symbol:   class Map                                                   
  location: class FlutterFreshchatPlugin                                
Note: /home/harsh/.pub-cache/hosted/pub.dartlang.org/flutter_freshchat-1.1.2/android/src/main/java/com/freshchat/flutter_freshchat/FlutterFreshchatPlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.   
```

Checked `FlutterFreshchatPlugin.java` , you forgot to import Map.

--- a/android/src/main/java/com/freshchat/flutter_freshchat/FlutterFreshchatPlugin.java
+++ b/android/src/main/java/com/freshchat/flutter_freshchat/FlutterFreshchatPlugin.java
@@ -4,6 +4,7 @@ import android.app.Activity;
 import android.util.Log;
 
 import java.util.ArrayList;
+import java.util.Map;
 
 import com.freshchat.consumer.sdk.FaqOptions;
 import com.freshchat.consumer.sdk.Freshchat;

